src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer<T extends Machine> implements MachineProducer {
```

Comment:
```suggestion
public class BulldozerProducer implements MachineProducer<Bulldozer> {
```

ми маємо параметризувати класс MachineProducer(те саме з іншими TruckProducer, ExcavatorProducer)